blib2to3/pytree.py

```diff
@@ -66,9 +93,13 @@ def __eq__(self, other):
             return NotImplemented
         return self._eq(other)
 
-    __hash__ = None  # For Py3 compatibility.
+    # __hash__ = None # For Py3 compatibility.
```

Comment:
Does mypyc choke on `__hash__ = None`? I'd be fine with just removing the line.

Reply:
mypy complains about it with a type error. I can fix it with a `# type: Any` though.